wg-naming/governance.md

```diff
@@ -0,0 +1,85 @@
+# WG Naming governance
+
+## Uphold project values
+
+WG Naming upholds the Kubernetes project's [core values](https://github.com/kubernetes/community/blob/master/values.md).
+
+### Inclusive is better than exclusive
+
+> Broadly successful and useful technology requires different perspectives and skill sets which can only be heard in a welcoming and respectful environment.
+
+This document covers some specific ways that respect and welcome look like in WG Naming.
+
+> Our community shows respect for the time and effort put into a discussion regardless of where a contributor is on their growth path.
+
+Like other SIGs and working groups, WG Naming has prerequisite skills, training, and experience that are necessary for meaningful contribution. For example:
+
+  - Education or professional experience in dismantling systemic discrimination
+  - Demonstrable familiarity with current antidiscrimination resources
+  - A track record of supporting inclusion in the Kubernetes project
+
+WG Naming has many stakeholders, with many people eager to add their voices. WG Naming invites folks who are new to antiracism or inclusive language, and folks less experienced with frameworks of discrimination to participate by observing and learning. WG Naming invites contributors who meet prerequisites to contribute within the scope of WG Naming’s goals.
+
+### Resources for antidiscrimination
+
+https://developers.google.com/style/inclusive-documentation
+https://www.antiracismresources.info/#online-resources
+How to Be An Antiracist, Ibrahim X. Kendi
+So You Want to Talk About Race, Ijeoma Oluo
```

Comment:
Can we add bullet points to these? Markdown renders this as one single line.

Reply:
```suggestion
- https://developers.google.com/style/inclusive-documentation
- https://www.antiracismresources.info/#online-resources
- _How to Be An Antiracist_, Ibrahim X. Kendi
- _So You Want to Talk About Race_, Ijeoma Oluo
```

For laziness

Reply:
Thanks :pray: